coach/resources/styles/components/concept-coach/launcher.less

```diff
@@ -42,6 +43,9 @@
         .justify-content(center);
         color: @dark-blue;
         h2 {
+          font-size: 2.6rem;
+          letter-spacing: -1px;
+          color: @dark-blue;
```

Comment:
i think this color would be better to fix this way: Connexions/webview#1490, otherwise PR looks good